src/test/java/com/meilisearch/sdk/http/CustomOkHttpClientTest.java

```diff
@@ -84,9 +84,9 @@ private String readBody(RequestBody body) {
 
     @Test
     void get() throws Exception {
-        BasicHttpRequest request =
-                new BasicHttpRequest(HttpMethod.GET, "/test", Collections.emptyMap(), "some body");
-        BasicHttpResponse response = (BasicHttpResponse) classToTest.get(request);
+        HttpRequest request =
+                new HttpRequest(HttpMethod.GET, "/test", Collections.emptyMap(), "some body");
+        HttpResponse response = (HttpResponse) classToTest.get(request);
```

Comment:
Is it required to cast to `HttpResponse` here? Because it is already a `HttpResponse`...